.gitignore

```diff
@@ -1,7 +1,5 @@
 /cache
 /data
-*.csv
-*.json
```

Comment:
I don't want to unignore all CSVs and JSON files. I'd rather commit them on a case by case basis (you can use `--force` to do that) when we do want to commit them.